Description: Ignore the 'Address family not supported by protocol' error that
 breaks the errorlines counting on buildds without a working IPv6 stack.
Origin: vendor
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2014-10-21
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -897,7 +897,7 @@
 fi
 
 # Error log messages
-count=`$GREP '^E ' /tmp/cups-$user/log/error_log | $GREP -v '(usb) crashed on signal 11' | $GREP -v '(dnssd) stopped with status 1' | $GREP -v 'loadFile failed: temp file: not a PDF file' | $GREP -v 'Failed to connect to system bus' | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' /tmp/cups-$user/log/error_log | $GREP -v '(usb) crashed on signal 11' | $GREP -v '(dnssd) stopped with status 1' | $GREP -v 'loadFile failed: temp file: not a PDF file' | $GREP -v 'Failed to connect to system bus' | $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
 	$GREP '^E ' /tmp/cups-$user/log/error_log
